src/trio/_sync.py

```diff
@@ -767,7 +780,11 @@ def acquire_nowait(self) -> None:
         return self._lock.acquire_nowait()
 
     async def acquire(self) -> None:
-        """Acquire the underlying lock, blocking if necessary."""
+        """Acquire the underlying lock, blocking if necessary.
+
+        Raises:
+          BrokenResourceError: if the owner of the lock exits without releasing.
```

Comment:
```suggestion
          BrokenResourceError: if the holder of the underlying lock exits without releasing it.
```

Feel free to ignore, but my personal preference is to say "holder" rather than "owner" of the lock, and in the case of Condition refer to the _underlying lock_ as in the class docstring.

Reply:
"owner" is publicly exposed in e.g. `LockStatistics` and already used in several other docstrings, so unless others agree I think it probably makes more sense to be consistent with that.
"underlying" sounds good though